network/test/epochtransition_test.go

```diff
@@ -175,38 +298,111 @@ func checkConnectivity(t *testing.T, mw *p2p.Middleware, ids flow.IdentityList)
 	}, 5*time.Second, time.Millisecond*100)
 }
 
-// sendMessagesAndVerify sends a message from each engine to the other engines
-// and verifies that all the messages are delivered.
-func sendMessagesAndVerify(t *testing.T, ids flow.IdentityList, engs []*MeshEngine, send ConduitSendWrapperFunc) {
+// assertDisconnected checks that the middleware of a node is not connected to any of the other nodes specified in the
+// ids list
+func assertDisconnected(t *testing.T, mw *p2p.Middleware, ids flow.IdentityList) {
```

Comment:
If this is only used in this test suite, could make it a method on the `MutableIdentityTableSuite ` struct. If it's more general we could move to `testUtil.go`